doc/user_guide/Using_Features_Offline.md

```diff
@@ -0,0 +1,114 @@
+# Using Features Offline
+
+## Creating Environment
+
+Under the hood, GeoViews features simply wrap ``cartopy`` features, so it's a matter of properly
+configuring ``cartopy`` ahead of time.
+
+1. Create a new cartopy environment (or use an existing one):
+
+    ```bash
+    conda create -n cartopy_env python=3.10
+    ```
+
+2. Install the required packages:
+
+    ```bash
+    conda install -c conda-forge geoviews cartopy cartopy_offlinedata
+    ```
+
+    Or if you have an environment already, you may just need [`cartopy_offlinedata`](https://anaconda.org/conda-forge/cartopy_offlinedata):
+
+    ```bash
+    conda install -c conda-forge cartopy_offlinedata
+    ```
+
+## Verifying Setup
+
+Now, we will verify that the shapefiles are available offline.
+
+1. Ensure offline shapefiles were downloaded:
+
+    ```python
+    from pathlib import Path
+    import cartopy
+
+    data_dir = Path(cartopy.config["pre_existing_data_dir"])
```

Comment:
I needed to use `cartopy.config["data_dir"]` to find some files. Also, make some sense with the environment variable set later.

Reply:
This should be the file directory where the cartopy_offlinedata was downloaded, so I'm wondering which files exactly are you missing?

Reply:
It works now for me, but I have no idea why :upside_down_face: